.gitlab-ci.yml

```diff
@@ -1364,11 +1362,14 @@ publish_docker_client_images:
 
 
 .template_release_docker_images:
+  only:
+    variables:
+      - ($BUILD_SERVERS == "true" && $BUILD_CLIENT == "true") || $RUN_INTEGRATION_TESTS == "true"
```

Comment:
Have you verified that this works?

Some time ago (~1 year ago) these variables expressions for only/except were not really working and I avoided them at all costs after some painful experiences. It might be that is fixed in GitLab, but I wanted to explicitly ask.

Reply:
Yes, I have verified them, they work!